bounties/packagist/pyrocms/pyrocms/2/vulnerability.json

```diff
@@ -0,0 +1,55 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-10-16",
+    "AffectedVersionRange": "*",
+    "Summary": "Cross-site Request Forgery (CSRF)",
+    "Contributor": {
+        "Discloser": "",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "packagist",
+        "Name": "pyrocms/pyrocms",
+        "URL": "https://packagist.org/packages/pyrocms/pyrocms",
+        "Downloads": "53,745"
+    },
+    "CWEs": [
+        {
+            "ID": "352",
+            "Description": "Cross-Site Request Forgery (CSRF)"
+        }
+    ],
+    "CVSS":
+    {
+        "Version": "3.1",
+        "AV": "N",
+        "AC": "L",
+        "PR": "H",
+        "UI": "N",
+        "S": "U",
+        "C": "N",
+        "I": "H",
+        "A": "N",
+        "E": "P",
+        "RL": "U",
+        "RC": "C",
+        "Score": "4.9"
```

Comment:
I would consider this impactful in this case: `CSRF` normally is `medium/high` ... we should set it to `6.1` to `7.5` :smile: